hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java

```diff
@@ -57,11 +57,11 @@ public class CreateBucketHandler extends BucketHandler {
               " user if not specified")
   private String ownerName;
 
-  enum AllowedBucketLayouts { FILE_SYSTEM_OPTIMIZED, OBJECT_STORE }
+  enum AllowedBucketLayouts { FILE_SYSTEM_OPTIMIZED, OBJECT_STORE, LEGACY }
```

Comment:
I agree to be mentioned LEGACY as defaultValue, but is that necessary to include `LEGACY` to the allowed list ?

Reply:
The method would throw an error if we do not include the default value in the allowed list. That's why Ibhave included it.